.examples/raspberrypi/docker-compose.yml

```diff
@@ -0,0 +1,42 @@
+version: "3.4"
+
+services:
+  app:
+    image: monica
+    env_file: .env
+    depends_on:
+      - db
+    ports:
+      - 8080:80
+    volumes:
+      - data:/var/www/html/storage
+    restart: always
+
+  cron:
+    image: monica
+    env_file: .env
+    restart: always
+    volumes:
+      - data:/var/www/html/storage
+    command: cron.sh
+    depends_on:
+      - db
+
+  db:
+    image: jsurf/rpi-mariadb:latest
+    ports:
+      - 3306:3306
```

Comment:
This should not be included in the example. It open mysql port for outside the container, which is not a good practice.
```suggestion
```

Reply:
Ah, yes, good catch—this was something I added in for some debugging on my end.